pkg/operator/openshift_sdn_test.go

```diff
@@ -1,39 +1,120 @@
-package stub
+package operator
 
 import (
 	"testing"
 
-	"github.com/openshift/openshift-network-operator/pkg/apis/networkoperator/v1"
+	netv1 "github.com/openshift/openshift-network-operator/pkg/apis/networkoperator/v1"
+	uns "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
 	. "github.com/onsi/gomega"
 )
 
-var OpenshiftSDNConfig = v1.NetworkConfig{
-	Spec: v1.NetworkConfigSpec{
-		DefaultNetwork: v1.DefaultNetworkDefinition{
-			Type: v1.NetworkTypeOpenshiftSDN,
-			OpenshiftSDNConfig: &v1.OpenshiftSDNConfig{
-				Mode: v1.SDNModePolicy,
+var OpenshiftSDNConfig = netv1.NetworkConfig{
+	Spec: netv1.NetworkConfigSpec{
+		ServiceNetwork: "172.30.0.0/16",
+		ClusterNetworks: []netv1.ClusterNetwork{
+			{
+				CIDR:             "10.128.0.0/15",
+				HostSubnetLength: 9,
+			},
+			{
+				CIDR:             "10.0.0.0/14",
+				HostSubnetLength: 8,
+			},
+		},
+		DefaultNetwork: netv1.DefaultNetworkDefinition{
+			Type: netv1.NetworkTypeOpenshiftSDN,
+			OpenshiftSDNConfig: &netv1.OpenshiftSDNConfig{
+				Mode: netv1.SDNModePolicy,
 			},
 		},
 	},
 }
 
 var manifestDir = "../../manifests"
 
+// TestRenderOpenshiftSDN has some simple rendering tests
 func TestRenderOpenshiftSDN(t *testing.T) {
 	g := NewGomegaWithT(t)
 
 	h := Handler{
-		config:      &OpenshiftSDNConfig,
+		config:      OpenshiftSDNConfig.DeepCopy(),
 		ManifestDir: manifestDir,
 	}
+	config := h.config.Spec
+	sdnConfig := config.DefaultNetwork.OpenshiftSDNConfig
 
-	errs := h.validateOpenshiftSDN(OpenshiftSDNConfig.Spec.DefaultNetwork.OpenshiftSDNConfig)
+	errs := h.validateOpenshiftSDN()
 	g.Expect(errs).To(HaveLen(0))
 
-	objs, err := h.renderOpenshiftSDN(OpenshiftSDNConfig.Spec.DefaultNetwork.OpenshiftSDNConfig)
+	_ = func(objs []*uns.Unstructured, kind, namespace, name string) {
```

Comment:
What is this? You're creating an anonymous function and then throwing it away without ever calling it?

Reply:
whoops, forgot to run GC.